middleware/http_signed_test.go

```diff
@@ -41,9 +41,9 @@ func TestHTTPSignedOnly(t *testing.T) {
 	assert.NoError(t, err)
 	rr := httptest.NewRecorder()
 	handler.ServeHTTP(rr, req)
-	assert.Equal(t, http.StatusBadRequest, rr.Code, "request without signature should fail")
+	assert.Equal(t, http.StatusForbidden, rr.Code, "request without signature should fail")
```

Comment:
feel like this should be StatusUnauthorized instead, as the request lacks valid authentication credentials for the target resource.